worker/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mediasoup-sys"
-version = "0.7.2"
+version = "0.7.3"
```

Comment:
This is a breaking API change between library and worker, please bump to 0.8.0 instead

Reply:
Done.